assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp/steps/CreateMessageStep.js

```diff
@@ -85,7 +86,11 @@ export default function CreateMessageStep() {
 		const { error } = await saveSettings();
 
 		if ( ! error ) {
-			navigateTo( dashboardURL );
+			navigateTo(
+				addQueryArgs( dashboardURL, {
+					notification: 'ad_blocking_recovery_setup_success',
+				} )
+			);
```

Comment:
Adding the query arg here will break the `isNavigatingTo( dashboardURL )` check above. Since this URL is only used for the success navigation, let's add the query arg in the `useSelect` and then rename `dashboardURL` to something like `setupSuccessURL`.

Reply:
Thanks for spotting this 👍